.github/workflows/publish-command.yml

```diff
@@ -128,45 +135,92 @@ jobs:
           source venv/bin/activate
           tox -r -c ./tools/tox_ci.ini
           pip install --quiet -e ./tools/ci_*
+      # Needed for Sentry steps
+      - name: Set Connector Name Env Variable
+        run: |
+          echo "CONNECTOR_IMAGE_NAME=$(echo ${{ github.event.inputs.connector }} | cut -d"/" -f2)" >> $GITHUB_ENV
       - name: Write Integration Test Credentials for ${{ github.event.inputs.connector }}
         run: |
           source venv/bin/activate
           ci_credentials ${{ github.event.inputs.connector }}
         env:
           GCP_GSM_CREDENTIALS: ${{ secrets.GCP_GSM_CREDENTIALS }}
-      - name: Set Name and Version Environment Vars
-        if: startsWith(github.event.inputs.connector, 'connectors')
+      - name: Bump ${{ github.event.inputs.connector }} Version
+        id: bump-version
+        env:
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+          FORCE_BUMP: ${{ github.event.inputs.force }}
+        run: |
+          ./tools/integrations/manage.sh bump_version airbyte-integrations/${{ github.event.inputs.connector }} ${{ github.event.inputs.bump-version }}
+      - name: Add Version Bump Success Comment
+        if: github.event.inputs.comment-id && success() && steps.bump-version.outputs.bumped == 'true'
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          comment-id: ${{ github.event.inputs.comment-id }}
+          body: |
+            > :rocket: Auto-bumped version for ${{github.event.inputs.connector}}
+      - name: Add Version Bump Failure Comment
+        if: github.event.inputs.comment-id && !success() && steps.bump-version.outputs.bumped == 'true'
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          comment-id: ${{ github.event.inputs.comment-id }}
+          body: |
+            > :x: Couldn't auto-bump version for ${{github.event.inputs.connector}}
+      - name: Build ${{ github.event.inputs.connector }}
         run: |
-          source tools/lib/lib.sh
-          DOCKERFILE=airbyte-integrations/${{ github.event.inputs.connector }}/Dockerfile
-          echo "IMAGE_NAME=$(echo ${{ github.event.inputs.connector }} | cut -d"/" -f2)" >> $GITHUB_ENV
-          echo "IMAGE_VERSION=$(_get_docker_image_version ${DOCKERFILE})" >> $GITHUB_ENV
+          ./tools/integrations/manage.sh build airbyte-integrations/${{ github.event.inputs.connector }}
+        id: build
+        env:
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+          # Oracle expects this variable to be set. Although usually present, this is not set by default on GitHub virtual runners.
+          TZ: UTC
+      - name: Test ${{ github.event.inputs.connector }}
+        if: github.event.inputs.run-tests == 'true'
+        run: |
+          ./tools/integrations/manage.sh test airbyte-integrations/${{ github.event.inputs.connector }}
       - name: Prepare Sentry
```

Comment:
I'm not super familiar with what a "Sentry Release" is, but I'm wondering if this stuff should all happen after the publish step runs and succeeds. Would we want a sentry release to be created before a publish step that coul potentially fail?

Reply:
I also don't really know what we are using sentry for - @Phlair have you come across any data in sentry lately?  Maybe we can remove this